tests/unit/lms/services/assignment_test.py

```diff
@@ -600,12 +129,10 @@ def noise():
     factories.Assignment(
         tool_consumer_instance_guid=MATCHING_TOOL_CONSUMER_INSTANCE_GUID,
         resource_link_id="noise_resource_link_id",
-        ext_lti_assignment_id="noise_ext_lti_assignment_id",
     )
     factories.Assignment(
         tool_consumer_instance_guid="noise_tool_consumer_instance_guid",
         resource_link_id=MATCHING_RESOURCE_LINK_ID,
-        ext_lti_assignment_id=MATCHING_EXT_LTI_ASSIGNMENT_ID,
     )
```

Comment:
Good lord these tests were horribly long.

There's something bugging me about these still looking at them, but I think that's a job for another PR. The `MATCHING_TOOL_CONSUMER_INSTANCE_GUID` stuff at the top seems a bit extra. I suspect we could create a fixture and then refer to the values from it.